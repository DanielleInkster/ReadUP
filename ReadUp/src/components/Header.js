import React from 'react';
import {View, Image, StyleSheet} from 'react-native';

const Header = () => {
  return (
    <View style={styles.header}>
      <Image style={styles.image} source={require('../images/title.png')} />
    </View>
  );
};

const styles = StyleSheet.create({
  header: {
    height: '12%',
    backgroundColor: '#0f4c75',
  },
  image: {
    padding: '3%',
    alignSelf: 'center',
    marginTop: '3%',
  },
});

export default Header;
