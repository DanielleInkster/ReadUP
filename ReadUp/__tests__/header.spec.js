import React from 'react';
import {render} from 'react-native-testing-library';
import Header from '../src/components/Header';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});

describe('Header', () => {
  it('displays the title', () => {
    const {queryByText} = render(<Header title="ReadUp" />);
    expect(queryByText('ReadUp')).not.toBeNull();
  });
});
