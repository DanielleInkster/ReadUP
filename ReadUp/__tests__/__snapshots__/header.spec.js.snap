// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#0f4c75",
      "height": "12%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/images/title.png",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "marginTop": "3%",
        "padding": "3%",
      }
    }
  />
</View>
`;
